lib/matplotlib/image.py

```diff
@@ -291,6 +291,15 @@ def get_size(self):
 
         return self._A.shape[:2]
 
+    def get_shape(self):
+        """
+        Return the shape of the image as tuple (numrows, numcols, channels).
+        """
+        if self._A is None:
```

Comment:
You could save having the RuntimeError repeated by making `get_size` call `get_shape`?

Reply:
> Left comments about testing, strongly prefer the exception gets tested but won't block merging over that.
> 
> I'm fine either way with merging the tests (but have enough of a preference to comment on it).

Thanks for the feedback, @jklymak @tacaswell. I decided to merge the tests and add the exception test.